README.md

```diff
@@ -318,6 +318,22 @@ _Analysis with custom headers_
 k8sgpt analyze --explain --custom-headers CustomHeaderKey:CustomHeaderValue
 ```
 
+_Print analysis stats_
+
+```
+k8sgpt analyze -s
```

Comment:
I can't see a world where you'd want to run this without capturing the output in some format. 
But for those that do, perhaps... `k8sgpt analyze --stats --no-output` ?

Reply:
This simply makes the result of the analysis unreadable with the stats.